README.md

```diff
@@ -229,16 +230,70 @@ Example output (plaintext):
 > Success!
 > ```
 
+### Development server
+
+OpenBttn contains a development server written in Go, it can be used for developing the web UI locally or for serving the OTA update for the SPWF01SA module.
+
+Installation via macOS:
+
+```
+$ brew install go
+```
+
+Usage:
+
+```
+go run cmd/openbttn/main.go -ip 192.168.0.123 -ota ./public/ota -public ./public/openbttn
+```
+
+`-ip` assumes button is at IP 192.168.0.123, used for redirecting some communication while developing the Web UI.
+
+`-ota` path to folder containing Wi-Fi module OTA image (more on this below)
+
+`-public` path to folder containing Web UI.
+
+The development server serves everything at `http://localhost:8774`.
+
+### OTA update of Wi-Fi module firmware
+
+#### Overview
+
+Depending on when your bttn was built, it may have shipped with an older Wi-Fi module firmware. It's recommended to upgrade this firmware, and the process is stable and safe, but as with all things, your mileage may vary.
+
+As of this writing, version 3.5 is available and brings many improvements over older versions.
+
+#### Updating the Wi-Fi module
+
+Download the latest Wi-Fi module firmware from [here](https://my.st.com/content/my_st_com/en/products/embedded-software/wireless-connectivity-software/stsw-wifi001.html) (requires signing up for a free account).
+
+Unzip it and locate the module firmware version you wish to use.
+
+For our example, we've chosen:
+> SPWF01S-160129-c5bf5ce-RELEASE-main.ota
+
+Move the firmware into the `ota` folder:
+
+```
+$ mv ~/Downloads/STSW-WIFI001/Rel.\ 3.5/OTA/SPWF01S-160129-c5bf5ce-RELEASE-main.ota public/ota/
+```
+
+Run the included web server (more on this in "Development server" above):
+
+```
+$ cd public && go run ../cmd/openbttn/main.go
+```
+
+Boot bttn in recovery mode, as described in the section above, and join the `OpenBttn` Wi-Fi network.
+
+Next, open a web browser to [http://192.168.1.1](http://192.168.1.1) and verify that the update URL shown matches what you expect, and that you can reach that URL yourself (doing so should trigger a file download).
+
+If it's working as expected, you may update the Wi-Fi module firmware. After it has succeeded, you may need to reconnect to the *OpenBttn* Wi-Fi access point to proceed with inputting your network settings.
```

Comment:
Maybe we could change `inputting` => `(re)configuring`? This because the OTA update will wipe the current network settings, if any.